frontend/src/components/editor/output/MarimoTracebackOutput.tsx

```diff
@@ -110,19 +113,16 @@ export const MarimoTracebackOutput = ({
             Fix with AI
           </Button>
         )}
-        {lastTracebackInfo && (
+        {tracebackInfo && !isWasm() && (
           <Button
             size="xs"
             variant="outline"
             onClick={() => {
-              const view = getCellEditorView(lastTracebackInfo.cellId);
-              if (view) {
-                insertDebuggerAtLine(view, lastTracebackInfo.lineNumber);
-              }
+              sendPdb({ cellId: tracebackInfo.cellId });
             }}
           >
             <BugPlayIcon className="h-3 w-3 mr-2" />
-            Insert breakpoint
+            Hook debugger
```

Comment:
Maybe "Re-run with debugger"

Reply:
Hmm, it it's actually rerunning though, just putting back where the failure happened.

maybe: "Launch debugger" ?

Reply:
Yea maybe Launch debugger is better then